src/Source.ruleset

```diff
@@ -4,6 +4,7 @@
     <Name Resource="MinimumRecommendedRules_Name" />
     <Description Resource="MinimumRecommendedRules_Description" />
   </Localization>
+  <Include Path="SecurityRulesEnabled.ruleset" Action="Default" />
```

Comment:
This file `Source.ruleset` is the set of rules that were already enabled on ML.NET. With this line change we now also enable the ones that are present on the `SecurityRulesEnabled` ruleset.

**NOTE**: Although the rules on this `Source.ruleset` file were always on ML.NET, since we didn't have the FxCopAnalyzer installed, then some of the rules on this file were pretty much ignored (since we didn't have any analyzer that would detect them). This is relevant, because now that we install the FxCop Analyzer these rules on this file will now be actually checked on ML.NET for the first time, and some of them raise errors. It's possible that we'll like to remove the rules on this file if this is the case, if these rules aren't enabled explicitly on the `SecurityRulesEnabled.ruleset` file.